index.js

```diff
@@ -24,6 +49,28 @@ const likes = document.querySelector("#profile .likes")
 likes.textContent = `${traveler.likes} Likes`
 
 
+const newForm = document.querySelector('form#new-animal-sighting-form')
+newForm.addEventListener('submit', function(event){
+    // console.log('form submitted', event.target)
+    event.preventDefault()
+    
+    const speciesInput = event.target.species.value
+    const videoInput = event.target.link.value
+    const photoInput = event.target.photo.value
+    const decsInput = event.target.description.value
+    const newObject = {
+        species: speciesInput,
+        video: videoInput,
+        photo: photoInput,
+        description: decsInput
+    }
+    renderAnimalSightingPost(newObject)
+})
```

Comment:
Great work on this last one!! If you wanted to get fancy w/ the ES6 syntax, you could make sure your variable names were the same names as the keys and do something like this...

```javascript
    const species = e.target["species"].value;
    const videoLink = e.target["animal-link"].value;
    const photo = e.target["animal-photo"].value;
    const description = e.target["sighting-description"].value;
    const newData = {
        species,
        videoLink,
        photo,
        description
    }
```
But I like your way and it's very readable and makes sense! Just showing you in case you see it done other ways.

PPS, what if we wanted to clear the form on a submit? There's probably a function on the event.target object that lets us do that... 🤔